interface/main/tabs/menu/menus/standard.json

```diff
@@ -1239,21 +1239,6 @@
             "global_req": [
```

Comment:
also need to make these menu changes in the file at sites/default/documents/custom_menus/Custom.json
(note that all you need to do is simply copy the interface/main/tabs/menu/menus/standard.json file to sites/default/documents/custom_menus/Custom.json)

Reply:
repost:
need to make these menu changes in the file at sites/default/documents/custom_menus/Custom.json
(note that all you need to do is simply copy the interface/main/tabs/menu/menus/standard.json file to sites/default/documents/custom_menus/Custom.json)